images/linux/scripts/installers/mysql.sh

```diff
@@ -9,17 +9,14 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
-
-# InstallMySQL database development files
-apt-get install libmysqlclient-dev -y
```

Comment:
Why do we delete libmysqlclient-dev?

Reply:
mysql-client will be installed in scope of mysql-server installation - it was verified. Thank you for the tip, libmysqlclient-dev definitely must be included into installation script. Fixed.